guides/theming.md

```diff
@@ -251,3 +251,27 @@ function changeTheme(themeName) {
 ### Theming your own components
 For more details about theming your own components,
 see [theming-your-components.md](./theming-your-components.md).
+
+### Showing strong focus indicators
+Angular Material has built-in support for showing **strong focus indicators** on all components. Focus indicators are visual markers that show users which element is focused. Angular Material's strong focus indicators render as highly visible outlines around elements.
```

Comment:
The sentence starting with "Focus indicators..." sounds like it's referring to focus indicators in general, not the strong ones. Maybe we should be more explicit?

Reply:
This is purposeful because by "focus indicators" I meant either Material's current focus styles or these new strong ones. Either way, they are visual markers that show users which element is focused. I was also back-and-forth on just removing this sentence if it is just going to cause more confusion.

Reply:
I ended up just removing this sentence.